src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -813,9 +855,12 @@ internal unsafe struct MethodTableAuxiliaryData
         private const uint enum_flag_HasCheckedCanCompareBitsOrUseFastGetHashCode = 0x0002;  // Whether we have checked the overridden Equals or GetHashCode
         private const uint enum_flag_CanCompareBitsOrUseFastGetHashCode = 0x0004;     // Is any field type or sub field type overridden Equals or GetHashCode
 
+        private const uint enum_flag_Initialized                = 0x0001;
         private const uint enum_flag_HasCheckedStreamOverride   = 0x0400;
         private const uint enum_flag_StreamOverriddenRead       = 0x0800;
         private const uint enum_flag_StreamOverriddenWrite      = 0x1000;
+        private const uint enum_flag_EnsuredInstanceActive      = 0x2000;
```

Comment:
This flag is trying to optimize the scenario as `enum_flag_HasModuleDependencies` (`enum_flag_HasModuleDependencies` was designed to be compatible with fragile ngen and multiple appdomains). We should delete `enum_flag_HasModuleDependencies` if we add this.

Reply:
Done.